### ISSUE TITLE
Issues running pFST

### ISSUE THREAD

**Issue by uod12345** [zeeev/vcflib#12]
Hi Zev, I'm trying to run pFST using your GPAT++ tool. It mentions using a comma separated list of inidividuals corresponding to the VCF file. I have more than 6000 individuals in mine. Is there a shorter way to do so? Something like {0:6000}? Do let me know.

**Comment by zeeev** [comment]
Hi @uod12345,

At this time there isn't a way shorthand way to do it. You'll probably hit a character limit on the command line. 

If you're comfortable with cpp, I'd suggest changing this function:

https://github.com/vcflib/vcflib/blob/master/src/pFst.cpp#L58

**Comment by uod12345** [comment]
Hi Zev,

I'm sorry, I'm not familiar with cpp; do I change it this way:

void loadIndices(map<int, int> & index, string set)  > void loadIndices(map<
1, 6000> & index, string set) ? # Include the numbers instead of int?

Also, I came across a blog post which gives a solution in this link :

https://www.biostars.org/p/305937/

I have not tried it yet. Do you think this will work?

Thank you for your reply.

Regards.

**Comment by zeeev** [comment]
Yes, something like that will work. I use to do something like `$(perl -e 'print join ",", (0..100)')`

**Comment by uod12345** [comment]
Thank you Zev! Will try these out.

Regards.

**Comment by zeeev** [comment]
Chances are you'll max out the binomial with 6000+ samples.
Consider downsampling or using wcFst.

**Comment by uod12345** [comment]
I have used wcFST to calculate FST on vcftools/plink. I wanted to use
GPAT++ to calculate P-values for those. I'm not sure of the effects of
downsampling. Will give it a shot.
